File: src/main/java/ru/citeck/idea/references/resolvers/XmlTagValueResolver.java
package ru.citeck.idea.references.resolvers;

import com.intellij.psi.xml.XmlTag;
import ru.citeck.idea.references.PsiElementValueResolver;

public class XmlTagValueResolver implements PsiElementValueResolver<XmlTag> {

    @Override
    public Class<XmlTag> getPsiElementType() {
        return XmlTag.class;
    }

    @Override
    public Object getValue(XmlTag psiElement) {
        return psiElement.getValue().getText();
    }

}

File: src/main/kotlin/ru/citeck/idea/references/resolvers/XmlTagValueResolver.kt
package ru.citeck.idea.references.resolvers

import com.intellij.psi.xml.XmlTag
import ru.citeck.idea.references.PsiElementValueResolver

class XmlTagValueResolver : PsiElementValueResolver<XmlTag> {

    override fun getPsiElementType(): Class<XmlTag> {
        return XmlTag::class.java
    }

    override fun getValue(psiElement: XmlTag): Any {
        return psiElement.value.text
    }
}
